.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,57 @@
+name: "godot-ci export"
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+env:
+  GODOT_VERSION: 4.2.2
+  EXPORT_NAME: sequence-escape
+
+jobs:
+  export-web:
+    name: Web Export
+    runs-on: ubuntu-20.04
+    container:
+      image: barichello/godot-ci:4.2.2
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+      - name: Setup
+        run: |
+          mkdir -v -p ~/.local/share/godot/export_templates/
+          mv /root/.local/share/godot/export_templates/${GODOT_VERSION}.stable ~/.local/share/godot/export_templates/${GODOT_VERSION}.stable
+      - name: Web Build
+        run: |
+          mkdir -v -p build/web
+          godot --headless --verbose --export-release "HTML5" ./build/web/index.html
+
+      # See: https://github.com/godotengine/godot-docs/issues/7084
+      - name: Workaround for SharedArrayBuffer support on GitHub Pages
+        run: |
+          cp build-aux/coi-serviceworker.js build/web
```

Comment:
The HTML5 export template adds:

```
<script src="coi-serviceworker.js"></script>
```

to the `<head>`. But I can't find a way to add random extra files to the exported directory.